app/chat/page.js

```diff
@@ -0,0 +1,19 @@
+'use client'
+
+import React from "react";
+import { useSearchParams } from 'next/navigation'
+import { Header } from '../components/Header'
+import { Footer } from '../components/Footer'
```

Comment:
I think we're not using this component, should we remove it from here? 👀

Reply:
Correcto, ya lo quite